src/eeeoh/eeeoh.ts

```diff
@@ -713,16 +721,34 @@ export const createOptions = <CustomLevels extends string>(
     mixin: (mergeObject, level, logger) => {
       const tier = getTier(mergeObject, level, logger);
 
+      let ddTags = {};
+
+      if (opts.eeeoh && base?.env && base.version) {
+        const config = getConfigForLogger(logger);
+        const { team } = config ?? opts.eeeoh;
+        const { env, version } = base;
+
+        ddTags = {
+          ddtags: ddtags({
+            env,
+            version,
+            ...(team ? { team } : {}),
+          }),
+        };
+      }
+
       return {
         ...original.mixin?.(mergeObject, level, logger),
-
         // Take precedence over the user-provided `mixin` for the `eeeoh` property
+        ...ddTags,
         ...formatOutput(tier),
       };
     },
 
     mixinMergeStrategy: (mergeObject, mixinObject) => {
-      const retain = 'eeeoh' in mixinObject ? { eeeoh: mixinObject.eeeoh } : {};
+      const eeeoh = 'eeeoh' in mixinObject ? { eeeoh: mixinObject.eeeoh } : {};
+      const ddTags =
+        'ddtags' in mixinObject ? { ddtags: mixinObject.ddtags } : {};
```

Comment:
Just to confirm, what's the current & expected behaviour if someone decides to do

```typescript
logger.info({ ddtags: ... }, 'msg')
```

And is it the same as if they were to do

```typescript
logger.info({ ddsource: ... }, 'msg')
```

```typescript
logger.info({ eeeoh: ... }, 'msg')
```

Reply:
Do you think a5287fdd2a1888d4e7dada31b9de7502cbd40b94 is equivalent? (Just confirming my understanding!)